service-discovery/git/src/main/scala/restui/servicediscovery/git/process/ProcessFlow.scala

```diff
@@ -0,0 +1,21 @@
+package restui.servicediscovery.git.process
+
+import scala.sys.process._
+import scala.util.Try
+
+import akka.NotUsed
+import akka.stream.scaladsl.Flow
+
+object ProcessFlow {
+
+  val flow: Flow[ProcessArgs, Try[List[String]], NotUsed] =
```

Comment:
This could be a type Alias just to give more meaning to the Response or write it in a different way. If F[T,F[F[A]],B]
It quiet complicate. 
Suggestion make Try to Empty List if its possible

Reply:
Yes, it's planned to refactor it 😉